rskj-core/src/main/java/org/ethereum/core/ImmutableTransaction.java

```diff
@@ -0,0 +1,28 @@
+package org.ethereum.core;
+
+import org.ethereum.crypto.ECKey;
+
+/**
+ * Created by ajlopez on 02/08/2017.
+ */
+public class ImmutableTransaction extends Transaction {

```

Comment:
Sounds like this adds overhead to the PR while not adding that much benefit. Though this change would be welcome.